adapters/jquery.js

```diff
@@ -157,10 +157,10 @@
 					editor.on( 'instanceReady', function( evt ) {
 						var editor = evt.editor;
 
-						setTimeout( function() {
+						setTimeout( function delayEditor() {
 							// Delay bit more if editor is still not ready.
 							if ( !editor.element ) {
-								setTimeout( arguments.callee, 100 );
+								setTimeout( delayEditor, 100 );
```

Comment:
```suggestion
								setTimeout( waitForEditor, 100 );
```

Since this code does the same thing as the one already refactored, you may go with the same naming so I suggest `waitForEditor` here too.